armi/reactor/blocks.py

```diff
@@ -2022,6 +2025,22 @@ def rotate(self, rad):
             in 60-degree increments (i.e., PI/6, PI/3, PI, 2 * PI/3, 5 * PI/6,
             and 2 * PI)
 
+        Notes
+        -----
+        Parameters marked with the ``rotatable``
+        :class:`~armi.reactor.parameters.parameterDefinitions.Category` will be rotated.
+        These parameters must adhere to one of the following rules to be rotated:
```

Comment:
Missing vector valued quantities like `displacementX` and `displacementY` where two block parameters form a vector. Alternatively, you could restructure `displacementX` and `displacmentY` to be a single parameter that follows these rules.

Reply:
Yes that was not called out in the docs. I've revamped the PR and kept the existing rotation routines (displacement, orientation) so those should be unimpacted